src/Lean/Widget/InteractiveCode.lean

```diff
@@ -24,24 +24,48 @@ structure SubexprInfo where
   subexprPos : Lean.SubExpr.Pos
   -- TODO(WN): add fields for semantic highlighting
   -- kind : Lsp.SymbolKind
+  /-- Additional tags for rendering subexpresions. This is used for the goal-diff system. -/
+  tags?: Option (Array String) := none
   deriving Inhabited, RpcEncodable
 
 /-- Pretty-printed syntax (usually but not necessarily an `Expr`) with embedded `Info`s. -/
 abbrev CodeWithInfos := TaggedText SubexprInfo
 
+/-- Map the given `tt : CodeWithInfos`, if its pos appears in `pm`
+then it will modify the info at that position using `merger`. -/
```

Comment:
```suggestion
```
This function is short enough and a little difficult to explain (I don't understand the docstring at all). I would just remove the docstring.